src/OpenGL/Extensions/Silk.NET.OpenGL.Extensions.ARB/ArbBindlessTexture.gen.cs

```diff
@@ -26,12 +26,6 @@ public unsafe partial class ArbBindlessTexture : NativeExtension<GL>
         [NativeApi(EntryPoint = "glGetImageHandleARB", Convention = CallingConvention.Winapi)]
         public partial ulong GetImageHandle([Flow(FlowDirection.In)] uint texture, [Flow(FlowDirection.In)] int level, [Flow(FlowDirection.In)] bool layered, [Flow(FlowDirection.In)] int layer, [Flow(FlowDirection.In)] PixelFormat format);
 
-        [NativeApi(EntryPoint = "glGetImageHandleARB", Convention = CallingConvention.Winapi)]
```

Comment:
Why have these been removed?

Reply:
seems to be related to the `Boolean` type in the 3rd argument? what is the point of `Boolean` vs `bool`, isnt `bool` just an alias for `Boolean`?

"The bool type keyword is an alias for the .NET [System.Boolean](https://learn.microsoft.com/en-us/dotnet/api/system.boolean) structure type that represents a Boolean value, which can be either true or false." -[docs](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/bool)

Reply:
Boolean is a GL Enum

Reply:
Upon further investigation, if any user is using these Boolean enums their code is broken so ok with removing these.